mantidproject/mantid
# Speed up SumOverlappingTubes

This speeds up the algorithm `SumOverlappingTubes` by removing a debugging statement and paralleling a couple of loops.

**To test:**

1. Compare the performance test speed for 6a5455a and d778d7c and check there is *big* a speed up.
1. Compare the performance test speed for 2a4eb13 and 4694f45 and check there is a *small* speed up.

Fixes #21512. 

**Release Notes** 
*Does not need to be in the release notes - changes to a recently introduced algorithm.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
